src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -16,9 +16,9 @@
  */
 public class PersonBuilder {
 
-    public static final String DEFAULT_NAME = "Alice Pauline";
+    public static final String DEFAULT_NAME = "Amy Bee";
     public static final String DEFAULT_PHONE = "85355255";
-    public static final String DEFAULT_EMAIL = "alice@gmail.com";
+    public static final String DEFAULT_EMAIL = "amy@gmail.com";
     public static final String DEFAULT_ADDRESS = "123, Jurong West Ave 6, #08-111";
```

Comment:
Sorry @CodyChew, it took me a while to get back to this PR.
The PR seems much simpler compared to the other two alternatives.
However, it's not immediately apparent why the default values here need to change. I assume these constants are used in other test cases as well?

Reply:
Yes @damithc, it is only failing in one test case in `AddCommandIntegrationTest`
```
public void execute_newPerson_success() {
        Person validPerson = new PersonBuilder().build();

        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
        expectedModel.addPerson(validPerson);

        assertCommandSuccess(new AddCommand(validPerson), model,
                String.format(AddCommand.MESSAGE_SUCCESS, validPerson), expectedModel);
}
```
This is because the `validPerson` uses the default name in `PersonBuilder.java` which is the same as the first person in the model which uses  `TypicalPersons.java`. This test case will work so long as the default name is not any name already in `getTypicalPersons()`

For reference:
```
public static List<Person> getTypicalPersons() {
        return new ArrayList<>(Arrays.asList(ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE));
}
```

Reply:
I think it's helpful to explain in the commit message why this change was made, so that future readers of the commit can know why this change was made without having to refer to this thread on GitHub. It can be something like:

```
Also, let's change the default values of.... This is because AddCommandIntegrationTest...
```

Reply:
Should I squash the commits into a single commit? and revise the commit message

Reply:
Sorry, missed this comment. Currently this change is in the same commit as the other changes to isSamePerson() right? They can remain in the same commit, with the modification to the commit message as what I typed above.